scss/components/_menu.scss

```diff
@@ -52,6 +52,11 @@ $menu-item-background-hover: $light-gray !default;
 
   li {
     @include disable-mouse-outline;
+
+    @if $global-flexbox {
```

Comment:
Remember flexbox is going to be primary from 6.4, and the space between in is more a bug with `inline-block`. I recommend we remove the margin and sit flush personally, as it would generally be better (and in 7 there would be no float anyway I expect!).